llama_stack/providers/inline/inference/sentence_transformers/sentence_transformers.py

```diff
@@ -34,14 +36,15 @@
 log = get_logger(name=__name__, category="inference")
 
 
-class SentenceTransformersInferenceImpl(
+class SentenceTransformersInferenceImpl(  # type: ignore[misc]
     OpenAIChatCompletionToLlamaStackMixin,
     OpenAICompletionToLlamaStackMixin,
     SentenceTransformerEmbeddingMixin,
     InferenceProvider,
     ModelsProtocolPrivate,
 ):
     __provider_id__: str
+    model_store: ModelStore | None = None  # type: ignore[assignment]
```

Comment:
please add details of why you need to ignore assignment

Reply:
this is already defined in `InferenceProvider` 

sorry my bad 

removed this 👍🏽